### ISSUE TITLE
Where to Download

### ISSUE THREAD

**Issue by TGN428** [ReneLergner/WPinternals#22]
Where can i Download WPInternals? # #

**Comment by hooddy** [comment]
https://github.com/ReneLergner/WPinternals/releases

**Comment by TGN428** [comment]
> 
> 
> https://github.com/ReneLergner/WPinternals/releases

But the Website is Down... Therefor i wrote it

**Comment by B3LiK0v** [comment]
Yup i want to unlock BL and do something with this WP L520 but no link nothing isn't available

**Comment by creativegamer03** [comment]
[https://web.archive.org/web/20201123212953/https://wpinternals.net/index.php/downloads](https://web.archive.org/web/20201123212953/https://wpinternals.net/index.php/downloads)
^ go there. its a snapshot of the download site.

**Comment by gus33000** [comment]
Hi,
you can download the latest version here: https://github.com/ReneLergner/WPinternals/releases/tag/2.9
